external-docs/docs/guides/errors/build-command-errors/attribute-not-linked.md

```diff
@@ -0,0 +1,11 @@
+# Error - Attribute Not Linked
+
+## When it occurs
+
+A units or attribute column is defined but it is not linked to any observation value column.
+
+## How to fix
+
+In the pivoted shape a unit or attribute column must be directly linked to a observation column. 
+
+<!-- TODO: Link to somewhere which helps the user define measures. -->
```

Comment:
Can you get rid of these todos? They don't make sense in the new files you've defined.

Reply:
Removed todos from each new documentation page.